components/select/select-option.tsx

```diff
@@ -84,8 +82,8 @@ const SelectOption: React.FC<React.PropsWithChildren<SelectOptionProps>> = ({
           max-width: 100%;
           justify-content: space-between;
           align-items: center;
-          font-weight: normal;
-          font-size: 0.75rem;
+          font-weight: 500;
+          font-size: 1rem;
```

Comment:
The font-size of select-option should be dynamic according to the size prop, same as the size prop to Select Component

Reply:
done